app/models/compute_resource.rb

```diff
@@ -6,6 +6,11 @@ class ComputeResource < ActiveRecord::Base
   include Parameterizable::ByIdName
   encrypts :password
 
+  attr_accessible :name, :provider, :description, :url, :display_type,
+      :set_console_password, :user, :password, :project,
+      :email, :location_ids, :location_names, :organization_ids,
```

Comment:
A thought - should we put attr_accessible for org_ids/names into Taxonomix instead?  It would force all taxable models to be mass assignment protected, but I don't think that's a bad thing.